core/client/queries/get-customer.ts

```diff
@@ -0,0 +1,23 @@
+import { getSessionCustomerAccessToken } from '~/auth';
+import { client } from '~/client';
+import { graphql, ResultOf } from '~/client/graphql';
+
+export const GetCustomerGroup = graphql(`
+  query CustomerGroup {
+    customer {
+      customerGroupId
+    }
+  }
+`);
+
+export type GetCustomerGroupResponse = ResultOf<typeof GetCustomerGroup>;
+
+export async function getCustomer() {
```

Comment:
This name is too generic for what it's doing. IMO we should name this `getCustomerGroupId` and return the customer group ID only